### DISCUSSION TITLE
Notes.t.id error in getAllNotes()

### DISCUSSION THREAD

**Discussion by tricaricom** [serverpod/serverpod#1782] [Q&A] [answered]
I am working through the tutorial "Build your first app" and the method `getAllNotes()` with the parameter `orderBy: Notes.t.id` is showing this error: The argument type 'ColumnInt' can't be assigned to the parameter type 'Column<dynamic> Function(NoteTable)?'.

Is anyone else seeing this?

**Comment by SandPod** [comment, marked as answer]
Hey @tricaricom!

Yes, unfortunately that is a documentation error reported by this [issue](https://github.com/serverpod/serverpod/issues/1772).

We will deploy a revised version of the documentation soon.

In the meantime, treat the `orderBy` parameter as a callback that passes the `t` referenced.

So the example
```
Future<List<Note>> getAllNotes(Session session) async {
  // By ordering by the id column, we always get the notes in the same order
  // and not in the order they were updated.
  return await Note.db.find(
    session,
    orderBy: Note.t.id,
  );
}
```

Should be:
```
Future<List<Note>> getAllNotes(Session session) async {
  // By ordering by the id column, we always get the notes in the same order
  // and not in the order they were updated.
  return await Note.db.find(
    session,
    orderBy: (t) => t.id,
  );
}
```
